src/Tools/SemanticSearch/BuildTask/GenerateFilteredReferenceAssembliesTask.cs

```diff
@@ -49,8 +50,7 @@ public sealed class GenerateFilteredReferenceAssembliesTask : Task
         \s*
         (?<Inclusion>[+|-]?)
         ((?<Kinds>[A-Za-z]+):)?
-        (?<MetadataName>[^#]*)
```

Comment:
Can't use `#` for comment in the metadata name since `#ctor` is document id for constructor.